src/component/parent/index.js

```diff
@@ -614,6 +619,17 @@ export class ParentComponent extends BaseComponent {
         };
     }
 
+    /*  Reposition
+     ----------
+
+       Reposition the child component window after Auto Resize
+     */
+
+    reposition() {
+        this.iframe.style.top = window.innerHeight / 2 - this.iframe.height / 2;
```

Comment:
Maybe we could just take all of this code:

https://github.com/vishakha94/xcomponent/blob/master/src/component/parent/drivers.js#L155-L175

And put it in a 'reposition' method, then call it from both places. Also it would be good to keep this in `drivers.js` since it's pretty specific to the iframe.

In general, as a rule of thumb, if there's any code where we're doing `if (this.iframe)` or `if (this.context === x)` it's best to put it in `drivers.js`, anything that's specific to iframes or popups or lightboxes should live there so parent.js doesn't have to deal with too much branching logic.